packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/Deployment/GithubPages/index.tsx

```diff
@@ -0,0 +1,134 @@
+import {
+  Button,
+  List,
+  ListAction,
+  Element,
+  ListItem,
+  Integration,
+  Link,
+  Stack,
+  Text,
+} from '@codesandbox/components';
+import track from '@codesandbox/common/lib/utils/analytics';
+import getTemplate from '@codesandbox/common/lib/templates';
+import React, { FunctionComponent, useEffect } from 'react';
+
+import { useAppState, useActions } from 'app/overmind';
+import { Info } from './Info';
+import { GitHubIcon, FileIcon, VisitIcon } from '../icons';
+
+export const GithubPages: FunctionComponent = () => {
+  const {
+    isLoadingGithub,
+    deployment: { deploying, githubSite },
+    user: {
+      integrations: { github },
+    },
+    editor: { currentSandbox },
+  } = useAppState();
+  const {
+    modalOpened,
+    signInGithubClicked,
+    deployment: { deployWithGitHubPages, fetchGithubSite },
+  } = useActions();
+
+  useEffect(() => {
+    fetchGithubSite();
+  }, [fetchGithubSite, currentSandbox.id]);
+
+  const template = getTemplate(currentSandbox.template);
+
+  if (
+    template.staticDeployment === false ||
+    template.githubPagesDeploy === false
+  ) {
+    return null;
+  }
+
+  const deploy = () => {
+    track('Deploy Clicked', { provider: 'github' });
```

Comment:
Nice that you made sure of tracking it! Will be nice to see once we deploy.

Reply:
Same!